README.md

```diff
@@ -114,11 +112,13 @@ This Project is presently under testing. But, pull requests are welcome.
 * Android App doesn’t connect to Windows Steam VR even after opening Phone App first and then SteamVR on windows
   1. Make sure that both the Windows and Android devices are in the same Local Network. (connected to the same router/wifi device)
   2. Sometimes, the port which PhoneVR use to connect win/mobile devices, also known as `Pairing Port (default :33333)`, might be used by other services on your devices(Windows/Android). Try changing the "Pairing Port" on `Android PhoneVR App` settings and `pairing_port` in `C:\Program Files\PhoneVR\pvrsettings.json` for `Windows PVR driver` and restart SteamVR. **Both Windows Pairing port and Android Pairing port should be the same**. Safe recommended port range : `30000 - 65535`
+  3. Incase above mentioned things have not been helpful to you, then you might want to **Install Debug variant of APK** from `[PhoneVR-0.1-beta_41-master.zip]/android/arm7/debug` which can help you/us to get more detailed logs.
```

Comment:
Moving this to a seperate bullet (after `Lag while using VR App on mobile`) would be much better because this does *not only* deals with *pairing failed* issues but with all general issues of app.

Reply:
Okay